code/src/main/java/com/onlinecontacttracing/servlets/get-data/GetCalendarDataForNegativeUser.java

```diff
@@ -0,0 +1,15 @@
+package com.onlinecontacttracing.authentication;
+
+import com.google.api.client.auth.oauth2.Credential;
+
+class GetCalendarDataForNegativeUser implements Runnable {
```

Comment:
I'm actually guessing there will be no difference in how you get the calendar data for a positive or negative user. Can you tell me what you think will differ to the point that it requires two different classes?

Reply:
There is, all the data for positive users is stored in a list and all the data for negative user is stored individually. This is so we can easily loop through a positive user's data and query a negative user's data